pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -97,7 +97,7 @@ func getPodInterfaceName(networks map[string]*v1.Network, cniNetworks map[string
 	}
 }
 
-func SetupNetworkInterfacesPhase1(vmi *v1.VirtualMachineInstance, pid int) error {
+func SetupPodNetworkPhase1(vmi *v1.VirtualMachineInstance, pid int) error {
```

Comment:
The idea of phase1 and phase2 runs deeper than this, in the code and in comments. I sense that eliminating them may be a more contentious matter. I see, for example, that @AlonaKaplan is suggesting another name. I would prefer to do it in a separate commit: I think that linking `NetworkInterfaces` and `PodNetwork` together is important on its own right, and should not hold until phases are gone.

Reply:
+1 for eliminating the phases completely in a future PR.
Just by calling the relevant functions in the virt-handler implies they are priv, there is no need to group them under this term.

Reply:
> The idea of phase1 and phase2 runs deeper than this, in the code and in comments. I sense that eliminating them may be a more contentious matter. I see, for example, that @AlonaKaplan is suggesting another name.

Yes, a name that focused on 'who' rather than on 'why' . If the goal of this PR is to help whoever reads the code to better understand what this does, I'd say the 'why' is important.

> I would prefer to do it in a separate commit: I think that linking `NetworkInterfaces` and `PodNetwork` together is important on its own right, and should not hold until phases are gone.

> +1 for eliminating the phases completely in a future PR.

I'm OK with that.

> Just by calling the relevant functions in the virt-handler implies they are priv, there is no need to group them under this term.

Yes, but that is not happening here - by renaming these functions accordingly, you can convey information to the reader of why this is like it is. That adds value to this intermediate state, until this is eventually refactored. 
And is so cheap to do it (rename from A to B costs the same independently of what B is) that I don't understand why we wouldn't want to do it.

Reply:
@maiqueb , it is impossible to explain it because it makes no sense.
I agree this change is not enough to make things better, but it is also not going into the wrong direction.
The name change was focusing on not confusing it with other things, which it partially did. Is to well describing what is going on? Sure it does not, no one can describe something that is doing a lot of things.

I think this change is not making things worse, it makes it a little bit less confusing without refactoring the core problem.

> Yes, but that is not happening here - by renaming these functions accordingly, you can convey information to the reader of why this is like it is. That adds value to this intermediate state, until this is eventually refactored.

I think the author is not interested to try and explain it, but rather try to avoid colliding the name with other similar functions.
If we want to get into the details, I am not really sure it just setups the privileged network details, as it also passes info from the virt-handler to the virt-launcher in a odd manner (files).

I find any attempt to make a good name here doomed. But I think the author did not try to give a name that expresses better that this phase1 does in the first place.

> And is so cheap to do it (rename from A to B costs the same independently of what B is) that I don't understand why we wouldn't want to do it.

I guess because not everyone agrees it is a better name that conveys well what this all-in-one function represents.

But if you manage to convince the author, I personally won't object to any naming... all are equally wrong in my view.